docs/getting-started/integration-method/vercelai.mdx

```diff
@@ -33,10 +33,27 @@ const openai = createOpenAI({
 });
 
 // Use openai to make API calls
-// For example:
-const response = await openai.chat.completions.create({
-  model: "gpt-3.5-turbo",
-  messages: [{ role: "user", content: "Hello, world" }],
+const response = streamText({
+  model: openai("gpt-4o"),
+  prompt: "Hello world",
+});
+```
+
+
+```javascript Anthropic
+import { createAnthropic } from "@ai-sdk/anthropic";
+
+const anthropic = createAnthropic({
+  baseURL: "https://anthropic.helicone.ai/v1",
```

Comment:
Pitfall: `/v1` suffix is required here

Vercel AI SDK requires that the baseURL contains this suffix, 
the `@anthropic-ai/sdk` requires that the URL does not contain this suffix!